tests/test_oml/test_models/test_visualisations.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Union
+
+import numpy as np
+import pytest
+from PIL import Image
+
+from oml.models import ResnetExtractor, ViTExtractor
+
+IMAGE_SIZE = (224, 224, 3)
+
+
+def get_image_pillow() -> Image.Image:
+    img = np.random.randint(0, 256, size=IMAGE_SIZE, dtype=np.uint8)
+    return Image.fromarray(img)
+
+
+def get_numpy_image() -> np.ndarray:
+    return np.random.randint(0, 256, size=IMAGE_SIZE)
+
+
+@pytest.mark.parametrize(
+    "draw_function",
+    [
+        ViTExtractor(None, "vits16", normalise_features=False).draw_attention,
+        ResnetExtractor(None, "resnet50", normalise_features=False, gem_p=None, remove_fc=False).draw_gradcam,
+    ],
+)
+@pytest.mark.parametrize(
+    "image",
+    [
+        get_numpy_image(),
+        get_image_pillow(),
+    ],
+)
+def test_visualisation(draw_function: Any, image: Union[np.ndarray, Image.Image]) -> None:
```

Comment:
Maybe it makes sense to assure that for the same image in `numpy` and `PIL` formats the result should be the same.

Reply:
I think it changes the logic of the test too much without a lof of additional value, because under the hood in the visualization functions everything is cast to numpy, so, there is nothing really to check

Reply:
As far as I know, taking into account the things under the hood of a function is not a correct way to write tests in this case. The implementation of the function may change in the future. Or maybe there will be some issues with other versions of numpy or PIL, so that they will produce different results as outputs of the function.

Maybe we can add another function to make such test.

Reply:
Basically, you are right, I was to lazy to accept it :) 

Well, I added one more test